react/features/base/media/components/web/VideoTrack.js

```diff
@@ -64,4 +69,22 @@ class VideoTrack extends AbstractVideoTrack<Props> {
     _onVideoPlaying: () => void;
 }
 
-export default connect()(VideoTrack);
+
+/**
+ * Maps (parts of) the Redux state to the associated VideoTracks props.
+ *
+ * @param {Object} state - The Redux state.
+ * @private
+ * @returns {{
+ *     noAutoPlayVideo: boolean
+ * }}
+ */
+function _mapStateToProps(state) {
+    const testingConfig = state['features/base/config'];
+
+    return {
+        noAutoPlayVideo: testingConfig && testingConfig.noAutoPlayVideo
```

Comment:
Isn't this going to be actually config.noAutoPlayVideo and not config.testing.noAutoPlayVideo?